src/Fixer/Phpdoc/PhpdocAlignFixer.php

```diff
@@ -102,6 +134,35 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    protected function createConfigurationDefinition()
+    {
+        $generator = new FixerOptionValidatorGenerator();
+
+        // By default, all tags apart from @property will be aligned for
+        // backwards compatibility
+        $defaultTagsToAlign = array(
```

Comment:
This variable is used only once, its value can be passed to `setDefault()` directly.